appengine/genetics/style.css

```diff
@@ -0,0 +1,171 @@
+#blockly {
+  position: fixed;
+  bottom: 10px;
+}
+
+#editor {
+  font-size: 16pt;
+  position: fixed !important;
+  height: auto !important;
+  bottom: 10px;
+  border: 1px solid #ddd;
+}
+
+#visualization {
+  position: relative;
+}
+
+#display {
+  border: 1px solid #ddd;
+  vertical-align: bottom;
+}
+
+#geneticsStatTable {
+  width: 406px;
+  max-width: 406px;
+  margin-left: -2px;
+  height: 20px;
+}
+
+#geneticsStatTable>tr>td {
+  padding: 0;
+  border: 1px solid;
+  background: #ccc;
+  border-radius: 2px;
+}
+
+#tabarea {
+  position: fixed !important;
```

Comment:
The CSS is based off of that in the pond-duck game. I'll have to take a look more closely to see whether it is needed.